.coveragerc

```diff
@@ -0,0 +1,8 @@
+[report]
```

Comment:
Can you put this in pyproject.toml since that's where we are going to end up and it saves proliferating lots of bespoke config files.

Reply:
Done. And I moved the config from `.flake8` to `setup.cfg`.

Reply:
I decided against moving to `setup.cfg` and chose to keep `.flake8` to keep in sync with the repo upstream.